packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -955,6 +957,38 @@ const BarfTurnTasks: GarboTask[] = [
     combat: new GarboStrategy(() => Macro.meatKill()),
     spendsTurn: () => globalOptions.target.attributes.includes("FREE"),
   },
+  {
+    name: "Liana Parachute",
+    ready: () =>
+      (sober() ||
+        (have($item`Drunkula's wineglass`) &&
+          canEquip($item`Drunkula's wineglass`))) &&
+      CrepeParachute.have() &&
+      shouldCheckParachute() &&
+      questStep("questL11Worship") > 3 &&
```

Comment:
I can see some flaws with this check but honestly I think the guy who encounters that problem deserves it